### PR TITLE
Enable ES2015 async/await by default in "ecmascript" package

### PR DESCRIPTION
**Pull request by avital**

### CODE REVIEW

File: packages/ecmascript-runtime/package.js [hunk 1]
```diff
@@ -7,17 +7,20 @@ Package.describe({
...
 
 Npm.depends({
-  "meteor-ecmascript-runtime": "0.2.6"
+  "meteor-ecmascript-runtime": "0.2.6",
+  "regenerator": "0.8.42"
 });
 
 Package.onUse(function(api) {
   api.use("modules");
+  api.use("promise");
```
Comment (benjamn) [inline packages/ecmascript-runtime/package.js]: Maybe comment that we're only depending on `promise` because regenerator needs it?

Comment (avital) [inline packages/ecmascript-runtime/package.js]: sure

**Comment by Slava** [comment]
yay?

**Comment by avital** [comment]
yay!

**Pull request merged**